qiskit-superstaq/qiskit_superstaq/superstaq_provider.py

```diff
@@ -316,13 +331,16 @@ def ibmq_compile(
         target: str = "ibmq_qasm_simulator",
         **kwargs: Any,
     ) -> qss.compiler_output.CompilerOutput:
-        """Returns pulse schedule(s) for the given circuit(s) and target.
+        """Returns pulse schedule(s) for the given qiskit circuit(s) and target.
+
         Args:
-            circuits: Qiskit QuantumCircuit(s)
-            target: String of target IBMQ device.
+            circuits: The circuit(s) to compile.
+            target: A string containing the name of a target IBM backend.
```

Comment:
nit: we should be consistent about using "IBMQ" instead of "IBM"
```suggestion
            target: A string containing the name of a target IBMQ backend.
```

Reply:
Done!